apps/js-machine-front/src/app/scenes/digest/Digest.tsx

```diff
@@ -82,7 +82,7 @@ export const Digest = memo(({ history, match }: Props) => {
     <Grid className={classes.root} container>
       <Hidden smDown>
         <Grid className={classes.leftSide} item md={3}>
-          <BackButton className={classes.backButton} onClick={history.goBack}>
+          <BackButton className={classes.backButton} onClick={ () => history.push('/news') }>
```

Comment:
please create method for it to avoid arrow function creation during each call of render method.
you can use useCallback function from React, see how its implemented in Admin app